UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -323,9 +316,20 @@ private void InitializeEnvironment()
                         key, resetParameters[key]
                     );
                 }
-
-                var pythonParameters = m_BrainBatcher.SendAcademyParameters(academyParameters);
-                Random.InitState(pythonParameters.Seed);
+                try
+                {
+                    var pythonParameters = m_BrainBatcher.SendAcademyParameters(academyParameters);
+                    Random.InitState(pythonParameters.Seed);
+                }
+                catch
+                {
+                    communicator = null;
```

Comment:
The batcher still has a reference to the (now defunct) communicator, right?

Reply:
Yes, you are right. I overwrite the batcher now.

Reply:
Sorry, missed that below.